src/vm-repair/azext_vm_repair/custom.py

```diff
@@ -180,16 +198,32 @@ def create(cmd, vm_name, resource_group_name, repair_password=None, repair_usern
         command.error_stack_trace = traceback.format_exc()
         command.error_message = str(exception)
         command.message = 'An unexpected error occurred. Try running again with the --debug flag to debug.'
+        
     finally:
+        # Execute commands on repair VM through run command option to unlock the copy of encrypted disk and mount.
+        REPAIR_DIR_NAME = 'azext_vm_repair'
+        SCRIPTS_DIR_NAME = 'scripts'
+        LINUX_RUN_SCRIPT_NAME = 'script.sh'
+        RUN_COMMAND_RUN_SHELL_ID = 'RunShellScript'
+        command_id = 'RunShellScript'
+        loader = pkgutil.get_loader(REPAIR_DIR_NAME)
+        mod = loader.load_module(REPAIR_DIR_NAME)
+        rootpath = os.path.dirname(mod.__file__)
+        run_script = os.path.join(rootpath, SCRIPTS_DIR_NAME, LINUX_RUN_SCRIPT_NAME)
+        repair_run_command = 'az vm run-command invoke -g {rg} -n {vm} --command-id {command_id} ' \
+                             '--scripts "@{run_script}" -o json' \
+                             .format(rg=repair_group_name, vm=repair_vm_name, command_id=command_id, run_script=run_script)
+        _call_az_command(repair_run_command)
         if command.error_stack_trace:
             logger.debug(command.error_stack_trace)
 
     # Generate return results depending on command state
-    if not command.is_status_success():
-        command.set_status_error()
-        return_dict = command.init_return_dict()
-        _clean_up_resources(repair_group_name, confirm=False)
-    else:
+    
+#    if not command.is_status_success():
+#        command.set_status_error()
+#        return_dict = command.init_return_dict()
+#        _clean_up_resources(repair_group_name, confirm=False)
+#    else:
         created_resources.append(copy_disk_id)
```

Comment:
I am not sure if the indent is correct.
And you can delete these code instead of mark it commented.